src/rosdiscover/launch.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+__all__ = ('Launch',)
+
+from types import MappingProxyType
+from typing import Any, Mapping, Sequence
+
+from loguru import logger
+import attr
+import yaml
+
+@attr.s(frozen=True, slots=True, auto_attribs=True)
+class Launch:
+    """
+    Attributes
+    ----------
+    filename: str
+        The file name of the launch files that should be launched
+    arguments: Mapping[str, str]
+        A set of launch arguments that should be passed to roslaunch
+    """
+    filename: str
+    arguments: Mapping[str, str]
+
+    def get_argv(self):
+        return list(map(lambda argk: f'{argk}:={self.arguments.get(argk)}', self.arguments.keys()))
```

Comment:
a list comprehension would be neater / easier to read here

Reply:
done